MHacks/v1/serializers/serializers.py

```diff
@@ -0,0 +1,76 @@
+from rest_framework import serializers
+from rest_framework.authtoken.serializers import AuthTokenSerializer
+from rest_framework.fields import CharField, ChoiceField
+from rest_framework.relations import PrimaryKeyRelatedField
+from rest_framework.serializers import ModelSerializer
+
+from MHacks.models import Event as EventModel
+from MHacks.models import Location as LocationModel
+from MHacks.models import Announcement as AnnouncementModel
+from MHacks.v1.serializers.util import UnixEpochDateField, DurationInSecondsField
+
+
+class MHacksModelSerializer(ModelSerializer):
+    def to_representation(self, instance):
+        if getattr(instance, 'deleted', False):
+            return {instance._meta.pk.name: str(instance.pk), 'deleted': True}
+        return super(MHacksModelSerializer, self).to_representation(instance)
+
+
+class AnnouncementSerializer(MHacksModelSerializer):
+    id = CharField(read_only=True)
+    broadcast_at = UnixEpochDateField()
+
+    class Meta:
+        model = AnnouncementModel
+        fields = ('id', 'title', 'info', 'broadcast_at', 'category', 'approved')
+
+
+class EventSerializer(MHacksModelSerializer):
```

Comment:
fyi biggest non-refactor change is using MHacksModelSerializer here instead of ModelSerializer